docs/2.guide/1.concepts/3.rendering.md

```diff
@@ -69,6 +69,28 @@ If you do use `ssr: false`, you should also place an HTML file in `~/app/spa-loa
 :read-more{title="SPA Loading Template" to="/docs/api/configuration/nuxt-config#spaloadingtemplate"}
 ::
 
+### SPA Mode
+
+Nuxt provides the ability to deploy your application into a [static hosting](getting-started/deployment#static-hosting). But when you use `ssr: false` and run `nuxi generate`, it will produce all the pages as static HTML files. In a common SPA we only need one single `index.html` file. In order to achieve this we can change how the routes are prerendered. Just add this to your hooks in your `nuxt.config.ts`:
```

Comment:
```suggestion
If you deploy your app to [static hosting](getting-started/deployment#static-hosting) with the `nuxi generate` or `nuxi build --prerender` commands, then by default, Nuxt will render every page as a separate static HTML file.

If you are using purely client-side rendering, then this might be unnecessary. You might only need a single `index.html` file, plus `200.html` and `404.html` fallbacks, which you can tell your static web host to serve up for all requests.

In order to achieve this we can change how the routes are prerendered. Just add this to your hooks in your `nuxt.config.ts`:
```